lib/datix/date_time.ex

```diff
@@ -42,22 +42,47 @@ defmodule Datix.DateTime do
     * `:am_pm_names` - a keyword list with the names of the period of the day,
       defaults to `[am: "am", pm: "pm"]`.
 
+    * `:pivot_year` - (since v0.2.0) a 2-digit year that represents the *pivot year* to use when
+      `%y` is used. `%y` represents a 2-digit year, but Datix doesn't assume anything
+      about which *century* such year refers to. For this reason, the `:pivot_year`
+      option is required whenever `%y` is present in the format string; if not
+      present, this function returns `{:error, :missing_pivot_year_option}`.
+      For example, if `pivot_year: 65`, then the 2-digit year `64` and lower will
+      refer to the current century (`2064` and so on at the time of writing this),
+      while the 2-digit year `65` and higher will refer to the previous century
+      (`1965` and so on).
+
+    * `:time_zone` - (since v0.3.0) TODO
```

Comment:
```suggestion
    * `:time_zone` - (since v0.3.0) a function that receives the parsed datetime as 
      `NaiveDateTime`, the zone abbreviation, and the offset to handle the time zone 
      and returns an `:ok` tuple with the `DateTime` or an `:error` tuple with 
      `Datix.ValidationError`. Defaults to a function handling only "UTC".
```

Reply:
@NickNeck 🤦 Sorry, missed that one. Thanks for filling it in!